CMakeLists.txt

```diff
@@ -34,10 +34,8 @@ include(cmake/Config.cmake)
 # https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html
 include(GNUInstallDirs)
 
-# add an option for choosing the build type (shared or static)
-if(NOT SFML_OS_ANDROID)
-    sfml_set_option(BUILD_SHARED_LIBS TRUE BOOL "TRUE to build SFML as shared libraries, FALSE to build it as static libraries")
-else()
+# disable static libraries on Android
+if(SFML_OS_ANDROID)
     set(BUILD_SHARED_LIBS TRUE)
 endif()
```

Comment:
https://github.com/SFML/SFML/commit/60894d1c1a5e51c6a02f85edbedf8ec91db1006b

I dug back in the commit history to find where this was added. I'm not totally sure why we have this here. Anything related to cross compilation belongs in a toolchain file. For what it's worth the Android CI jobs still succeed even if we remove this and build them as static libraries. Hopefully someone can point me to some Android docs explaining this requirement so I can add that link to the comment.

Reply:
@intjelic Do you know if we still need this?